### PR TITLE
2 update example for alttester v1.8.0

### PR DESCRIPTION
**Pull request by SandyEma9**

### CODE REVIEW

**Review by iulianatodoran — APPROVED** [review]
Tested locally and everything is ok. 
LGTM.

**Comment by SandyEma9** [comment]
@robert-altom @RobertPoienar @gomboskriszta15 @ReichEdvin

File: appium.log [hunk 1]
```diff
@@ -0,0 +1,574 @@
+2022-03-10 06:57:49:005 - [Appium] Welcome to Appium v1.22.2
```
Comment (robert-altom) [inline appium.log]: Remove the logs form the repository.

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+.idea/
+
+# VS Code
+.vscode/
+
+# MacOS
+.DS_Store
+
+# AltUnityTester
+AltUnityTesterLog.txt
```
Comment (robert-altom) [inline .gitignore]: Here you can update this to: 

```
# AltTester Logs 
*.logs
```

File: tests/pages/base_page.py [hunk 3]
```diff
@@ -3,7 +3,9 @@
 
 sys.path.append(os.path.dirname(__file__))
```
Comment (robert-altom) [inline tests/pages/base_page.py]: I think you can remove all calls to `sys.path.append(os.path.dirname(__file__))`.

File: requirements.txt [hunk 4]
```diff
@@ -1,8 +1,8 @@
+AltTester-Driver==1.8.0
 selenium
 pytest
 pytest-selenium
```
Comment (robert-altom) [inline requirements.txt]: I don't think we use `selenium`, `pytest-selenium`, `pytest-html`, or `assertpy` so we should remove them from `requirements.txt`.

File: tests/test_start_page.py [hunk 5]
```diff
@@ -0,0 +1,18 @@
+from .utils import AltTestCase
```
Comment (rucindrea) [inline tests/test_start_page.py]: why are we calling this file `utils`? it should be alt_test_case or something like this.

**Comment by rucindrea** [comment]
LGTM

**Pull request merged**